File: CompilationTest/StrLenInferIssue/results/StrLenInferIssue.java
class StrLenInferIssue {
    public static Integer test(String str) throws Exception
    {
        return str.length();
    }
}

class Program {
    public static void main(String[] args) throws Exception {
        System.out.println(StrLenInferIssue.test("hello"));
    }
}

File: CompilationTest/StrLenInferIssue/results/StrLenInferIssue.cs
using System;

public class StrLenInferIssue
{
    public static int Test(string str)
    {
        return str.Length;
    }
}

public class Program
{
    static public void Main(string[] args)
    {
        Console.WriteLine(StrLenInferIssue.Test("hello"));
    }
}